### ISSUE TITLE
[bug] <rh-subnav> keyboard nav should go to active item

### ISSUE THREAD

**Issue by nikkimk** [RedHat-UX/red-hat-design-system#769]
### Describe the bug

By default, when tabbing into <rh-subnav>, initial focus is set at first item instead of active item.

### Which Component?

rh-subnav

### Steps to reproduce

1. Go to https://deploy-preview-638--red-hat-design-system.netlify.app/elements/footer/code/
2. Click tab to enter subnav
3. The focus is set to the first item (Overview) instead of the active item (Code).

### Expected behaviour

The focus should be set to the active item (Code)

### Element HTML

```HTML
<rh-subnav>
    <a href="/elements/footer/" class="first" tabindex="0">Overview</a>
    <a href="/elements/footer/styles/" tabindex="-1">Styles</a>
    <a href="/elements/footer/guidelines/" tabindex="-1">Guidelines</a>
    <a active="" href="/elements/footer/code/" tabindex="-1">Code</a>
    <a href="/elements/footer/accessibility/" class="last" tabindex="-1">Accessibility</a>
</rh-subnav>
```

### Operating System (OS)

macOS
